std/parallelism.d

```diff
@@ -2889,11 +2889,11 @@ public:
             }
         }
 
-        ref T opIndex(size_t index)
+        ref opIndex(this Qualified)(size_t index)
         {
             import std.conv : text;
             assert(index < size, text(index, '\t', uint.max));
-            return *(cast(T*) (data + elemSize * index));
+            return *(cast(CopyTypeQualifiers!(Qualified, T)*) (data + elemSize * index));
```

Comment:
An imperfect name - redundant, should be `CopyQualifiers` because qualifiers only apply to types. cc @Hackerpilot